Userland/Utilities/run-tests.cpp

```diff
@@ -18,7 +18,6 @@
 namespace Test {
 TestRunner* TestRunner::s_the = nullptr;
 }
-using enum Test::Modifier;
```

Comment:
Big sad :(

Reply:
Yup. Clang 13 is probably not out until late this year, so no chance to do it with a somewhat vanilla Clang.

Maybe if I'm finished with getting the build working, I could look into backporting the recent commits that added this feature.